src/components/StaticContent/HowItWorks.re

```diff
@@ -66,7 +66,7 @@ module HowItWorks = {
                     <S> "As the gaurdian of a wildcard, you pay a " </S>
                     <strong> <S> "Harberger tax" </S> </strong>
                     <S>
-                      " (percentage based fee) on your newly set sale price. Your harberger tax goes towards supporting conservation. "
+                      " (percentage based fee) on your newly set sale price. Your harberger tax goes towards supporting the associated conservation organisation. "
```

Comment:
Hmm, I think this is good, but subtly inaccurate maybe since some animals don't (won't) have a specific organisation.
```suggestion
                      " (percentage based fee) on your newly set sale price. This fee goes towards supporting wildlife conservation. "
```

I also wonder if we should avoid using the word harberger tax when context makes it unnecessary.

Reply:
I like your simpler wording. I think all animals will eventually link to an organization and it makes sense in the context, but agree could be subtly misleading.